src/ec/packages/ec-component-tag/adapter.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable import/no-extraneous-dependencies, no-param-reassign */
+const adapter = initialData => {
+  const adaptedData = JSON.parse(JSON.stringify(initialData));
+  const tag = { tag: adaptedData };
+
+  if (adaptedData.dismissButtonLabel) {
+    tag.default_icon_path = '/icons.svg';
+    tag.tag.type = 'removable';
+    tag.extra_attributes = [
+      { name: 'aria-label', value: adaptedData.dismissButtonLabel },
```

Comment:
Maybe best to use the existing aria_label parameter here and set `tag.tag.aria_label` instead?

Reply:
i'll take a look, thanks for reviewing this.. ;)

Reply:
Absolutely right...thanks ;)